osquery/events/linux/auditeventpublisher.cpp

```diff
@@ -419,6 +434,21 @@ bool GetStringFieldFromMap(std::string& value,
   return true;
 }
 
+bool GetStringFieldFromIntMap(
```

Comment:
This function is only used by the seccomp table, can it be moved inside an anonymous namespace in seccomp_events.cpp? (Example: isPublisherEnabled in auditeventpublisher.cpp)